code/modules/clothing/under/_under.dm

```diff
@@ -337,6 +338,14 @@
 		if(H.w_uniform == src)
 			H.update_suit_sensors()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	. = ..()
+	if(!can_toggle_sensors(user))
+		return
+
+	src.sensor_mode = SENSOR_COORDS
+	balloon_alert(user, "your suit will now report your exact vital lifesigns as well as your coordinate position.")
```

Comment:
A balloon alert:
- should not be longer than five words 
- should not end in punctuation except for `...` and `!` and neither of these fit this scenario